ts-sdk/whirlpool/tests/sysvar.test.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, it, beforeAll } from "vitest";
+import type { SysvarRent } from "@solana/sysvars";
+import { lamports } from "@solana/web3.js";
+import { calculateMinimumBalance } from "../src/sysvar";
+import assert from "assert";
+import { setupAta, setupMint } from "./utils/token";
+import { fetchMint } from "@solana-program/token-2022";
+import { rpc } from "./utils/mockRpc";
+import { getTokenSizeForMint } from "../src/token";
+
+describe("Sysvar", () => {
+  let rent: SysvarRent;
+
+  beforeAll(async () => {
+    rent = {
```

Comment:
This doesn't necessarily have to be in the beforeAll (since it is not asynchronous). You can just init the rent directly as a const